ci/kokoro/docker/build-in-docker-cmake.sh

```diff
@@ -118,6 +118,11 @@ ${CMAKE_COMMAND} \
 echo
 io::log_yellow "Finished CMake config"
 
+if [[ "${CLANG_TIDY:-}" = "yes" ]]; then
+  io::log_yellow "clang-tidy config:"
+  clang-tidy -dump-config
```

Comment:
fwiw, this dumps over 300 lines of config output on all `clang-tidy` builds. An alternative might be to run `ci/kokoro/docker/build.sh clang-tidy clang-tidy --dump-config` when we need to see the output exactly as the build would see it.